cluster/pulumi/common/src/auth0/secrets.ts

```diff
@@ -151,7 +151,9 @@ export function installAuth0UiSecretWithClientId(
     },
     {
       dependsOn: xns.ns,
-      aliases: clientName ? [`splice-auth0-ui-secret-${xns.logicalName}-${clientName}`] : [],
+      aliases: clientName
+        ? [{ name: `splice-auth0-ui-secret-${xns.logicalName}-${clientName}` }]
```

Comment:
do we have some docs why this is correct? we did always use the regular format right?

Reply:
ah i guess we had apis built on top. lgtm